conan/tools/cmake/cmakedeps/templates/config.py

```diff
@@ -67,11 +68,19 @@ def template(self):
             endif()
         endforeach()
 
+        {% if legacy_variable_prefix %}
+        set({{ legacy_variable_prefix }}_VERSION_STRING "{{ version }}")
+        set({{ legacy_variable_prefix }}_INCLUDE_DIRS {{ pkg_var(pkg_name, 'INCLUDE_DIRS', config_suffix) }} )
+        set({{ legacy_variable_prefix }}_INCLUDE_DIR {{ pkg_var(pkg_name, 'INCLUDE_DIRS', config_suffix) }} )
+        set({{ legacy_variable_prefix }}_LIBRARIES {{ pkg_var(pkg_name, 'LIBRARIES', config_suffix) }} )
+        set({{ legacy_variable_prefix }}_DEFINITIONS {{ pkg_var(pkg_name, 'DEFINITIONS', config_suffix) }} )
+        {% else %}
         set({{ file_name }}_VERSION_STRING "{{ version }}")
         set({{ file_name }}_INCLUDE_DIRS {{ pkg_var(pkg_name, 'INCLUDE_DIRS', config_suffix) }} )
         set({{ file_name }}_INCLUDE_DIR {{ pkg_var(pkg_name, 'INCLUDE_DIRS', config_suffix) }} )
         set({{ file_name }}_LIBRARIES {{ pkg_var(pkg_name, 'LIBRARIES', config_suffix) }} )
         set({{ file_name }}_DEFINITIONS {{ pkg_var(pkg_name, 'DEFINITIONS', config_suffix) }} )
+        {% endif %}
```

Comment:
this can be made more compact, It's not necessary to repeat the templates with an if...else